### PR TITLE
log to stdout when being run from jenkins

### PR DESCRIPTION
**Pull request by wjimenez5271**
I've been running GD from Jenkins instead of cron for a number reasons, most notably so the team can have visibility into its activity and trigger runs ad-hoc.

This might not be the most elegant way to resolve this, so feel free to suggest a better way to go about it. Was trying to find a solution that wouldn't affect how people are using it already and wouldn't require extra config.

### CODE REVIEW

File: lib/grocery_delivery/logging.rb [hunk 1]
```diff
@@ -39,7 +39,9 @@ def self.logit(level, msg)
       # `%` in `msg` then ruby will interpret it as a printf string and
       # expect more arguments to log().
       Syslog.log(level, '%s', msg)
-      puts msg if $stdout.tty?
+      # Assuming if env variable 'BUILD NUMBER' exists we're being run
```
Comment (jaymzh) [inline lib/grocery_delivery/logging.rb]: Thats... an odd assumption. Should we add a --stdout option?

**Comment by jaymzh** [comment]
ping?

**Comment by wjimenez5271** [comment]
Sorry, meant to reply. `--stdout` is fine by me, I was just trying to localize this, but thats probably the right way to do it. I'll update the PR with that soon.

**Comment by jaymzh** [comment]
Ping?

**Comment by irabinovitch** [comment]
We made some similar changes as we have a jenkins based workflow as well.  --stdout sounds good to me.

**Comment by jaymzh** [comment]
@wjimenez5271 ping?

**Comment by wjimenez5271** [comment]
Sorry, I haven't forgotten, just got swamped with stuff. I'll try to have a PR for this early in the week

**Comment by jaymzh** [comment]
@wjimenez5271 @irabinovitch beat ya to it. :)

**Comment by wjimenez5271** [comment]
nice :-)

**Pull request closed without merging**